apps/src/code-studio/pd/workshop_dashboard/components/survey_results/free_response_section.story.jsx

```diff
@@ -1,47 +1,43 @@
 import React from 'react';
 import FreeResponseSection from './free_response_section';
-import reactBootstrapStoryDecorator from '../../../reactBootstrapStoryDecorator';
```

Comment:
This decorator is just a div wrapping the component with an id, so I did this manually below (decorators are not supported in the new storybook)